frontend/src/service.json

```diff
@@ -133,44 +149,50 @@
         "url": "https://suggestit.chalmers.it",
         "description": "Submit suggestions for Hubben 2.1 or the IT student division.",
         "github_url": "https://github.com/cthit/SuggestIT",
-        "fontawesome_icon": "fa-lightbulb-o"
+        "fontawesome_icon": "fa-lightbulb-o",
+        "category": "Hubben"
     },
     
     {
         "title": "BussIT",
         "url": "https://bussit.chalmers.it",
         "description": "A quick way to know when the buses and trams leave Chalmers.",
         "github_url": "https://github.com/cthit/BussIT",
-        "fontawesome_icon": "fa-bus"
+        "fontawesome_icon": "fa-bus",
+        "category": "Information"
     },
     
     {
         "title": "VulcanTV",
         "url": "https://vulcantv.chalmers.it",
         "description": "Remotely control the TV in Hubben 2.1.",
         "github_url": "https://github.com/cthit/goVulcanTV",
-        "fontawesome_icon": "fa-tv"
+        "fontawesome_icon": "fa-tv",
+        "category": "Hubben"
     },
     
     {
         "title": "LinkIT",
         "url": "https://linkit.chalmers.it",
         "description": "Link shortener for the Software Engineering Student Division (IT).",
         "github_url": "https://github.com/cthit/LinkIT",
-        "fontawesome_icon": "fa-link"
+        "fontawesome_icon": "fa-link",
+        "category": "Tools"
     },
     {
         "title": "SongBook",
         "url": "https://songbook.chalmers.it",
         "description": "The official digital songbook of the Software Engineering Student Division (IT).",
         "github_url": "https://github.com/cthit/songbook",
-        "fontawesome_icon": "fa-music"
+        "fontawesome_icon": "fa-music",
+        "category": "Hubben"
     },
     {
         "title": "hexIT",
         "url": "https://hex.chalmers.it/",
         "description": "The official color palette for IT",
```

Comment:
I know this isn't changed in this PR but 09cdda is not an official hexcolor for the IT section. There are none actually